src/met-atlas-viewer.js

```diff
@@ -39,130 +43,102 @@ import { makeIndexSprite } from './helpers';
  *     viewer should be placed.
  * @returns {Object} A control object with functions for controlling the viewer.
  */
-function MetAtlasViewer(targetElement) {
-  const container = document.getElementById(targetElement)
+const MetAtlasViewer = (targetElement) => {
+  const container = document.getElementById(targetElement);
 
   // Camera variables
   let fieldOfView = 90;
   let aspect = container.offsetWidth / container.offsetHeight;
   let near = 1;
   let far = 10000;
-  var camera = new PerspectiveCamera(fieldOfView, aspect, near, far)
+  let camera = new PerspectiveCamera(fieldOfView, aspect, near, far);
   camera.position.z = 3000;
   let nodeSelectCallback, updateCameraCallback;
 
-  var cameraDefault = {
+  const cameraDefault = {
     position: Object.assign({}, camera.position),
     up: Object.assign({}, camera.up),
-    target: Object.assign({}, camera.up)
+    target: Object.assign({}, camera.up),
   };
-  var flyTarget = {
+  const flyTarget = {
     active: false,
     start: undefined,
     startUp: undefined,
     end: undefined,
     target: undefined,
     startTime: undefined,
     targetUp: undefined,
-    runTime: 750
+    runTime: 750,
   };
 
-  // Set default colors
-  var nodeDefaultColor = [255,255,255];
-  var connectionStartColor = [0, 127, 255];
-  var connectionEndColor = [0, 127, 0];
-  var nodeSelectColor = [255, 0, 0];
-  var connectionSelectColor = [255, 255, 0];
-  var hoverSelectColor = [255, 0, 255];
-  var hoverConnectionColor = [255, 0, 0];
-
   // Create the scene and set background
-  var scene = new Scene();
-  scene.background = new Color( 0xdddddd );
+  let scene = new Scene();
+  scene.background = new Color(defaultColors.sceneBackgroundColor);
 
   // Create color picking scene and target
-  var indexScene = new Scene();
-  indexScene.background = new Color( 0xffffff );
-  var indexTarget = new WebGLRenderTarget(1,1);
+  const indexScene = new Scene();
+  indexScene.background = new Color(defaultColors.indexSceneBackgroundColor);
+  const indexTarget = new WebGLRenderTarget(1, 1);
 
   // Create object group for the graph
-  var graph = new Group();
+  let graph = new Group();
 
   // Create nodeMesh and connectionMesh as globals so that we can modify them
   // later
-  var nodeMesh;
-  var connectionMesh;
+  let nodeMesh;
```

Comment:
I don't think this is needed. Is there any best practice about if variables should always be initialized here?

Reply:
It is a good practice to initialize variables as mentioned at https://www.w3schools.com/js/js_best_practices.asp
But it's up to you to implement it or not.

Reply:
Thanks for the link @nanjiangshu! I was reading about it on [mdn docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) and didn't see any recommendations there. I also think having `undefined` as an initial value is not helpful since it conveys nothing about the intended use so I think we can leave this as is for now.